.github/workflows/reference-implementation.yml

```diff
@@ -51,18 +51,20 @@ jobs:
       run: | 
         npm install --prefix tests/packages/openactive-broker-microservice
         npm install --prefix tests/packages/openactive-integration-tests
+    - name: Run Checks on the Code (Test the Tests!)
+      run: npm test --prefix tests
```

Comment:
Based on your other comment "`npm test --prefix tests` would not run anything", what does this do? I haven't tried the command myself...

Reply:
`tests` is just the directory into which openactive-test-suite is git cloned in the instance that the GH action spawns

Reply:
Doh sorry, of course!